### DISCUSSION TITLE
handbook/enterprise_security/devsecops/process_and_landing/security_review/

### DISCUSSION THREAD

**Discussion by giscus** [SEC-CAFE/handbook#4] [General]
# handbook/enterprise_security/devsecops/process_and_landing/security_review/

软件安全研发流程中需求设计阶段的安全要求与评审指引、实践

https://sec.cafe/handbook/enterprise_security/devsecops/process_and_landing/security_review/

**Comment by user1430** [comment]
对安全评审的建议：
1. 长checklist给到任何人都会看的头痛，如果有意向进行评审工作的实际驱动，不妨进行场景化的评审，针对业务场景给出控制在20条最好10条左右的checklist是能够广泛落地和广受好评的操作；
2. DFD极少见到业务会画，尝试自动翻译架构图和时序图成DFD吧，能给到完整场景描述的时序图已经是一个好产品了
